src/constants/envs.js

```diff
@@ -42,15 +42,16 @@ const getDefaultENVs = cliRootDir => {
 
   // Build the path to the global defaults env
   const globalDefEnv = path.join(GLOBAL_CONFIG_FOLDER, '/', DEFAULT_ENV)
+  const cliRootDefEnv = path.join(CLI_ROOT_CONFIG_FOLDER, DEFAULT_ENV)
```

Comment:
This seems to now be overly complicated, due to there being two possible roots for the `globalConfig` folder

Reply:
updated back to just one